File: resolucoes/JavaScript/lista01/ex011.js
const prompt = require('prompt-sync')();

function main() {

const a = Number(prompt("Digite o valor de A:"))
const b = Number(prompt("Digite o valor de B:"))
const c = Number(prompt("Digite o valor de C:"))

const delta = (b * b) - (4 * a * c)

console.log(`O valor de Delta é: ${delta}`)
}

module.exports = {
    main
}

File: resolucoes/kotlin/lista01/src/main/kotlin/ex011.kt
import java.util.*

internal fun calcularDelta() {
    val scanner = Scanner(System.`in`)
    print("Digite o valor de A: ")
    val a = scanner.nextDouble()
    print("Digite o valor de B: ")
    val b = scanner.nextDouble()
    print("Digite o valor de C: ")
    val c = scanner.nextDouble()

    // Calcula o valor de Delta
    val delta = (b * b) - (4 * a * c)

    // Exibe o valor de Delta
    println("O valor de Delta é: $delta")
}

fun main() {
    calcularDelta()
}
